### DISCUSSION TITLE
Use with vue 3

### DISCUSSION THREAD

**Discussion by GaryJurgens** [ericblade/quagga2#384] [General]
Hi,

Thank you for a great library,

Please pardon my ignorance, but is there an example for implementing this with Vue 3 (or quasar), targeted to Quagga2

I have searched, but I cannot see how to use it. 

I would appreciate if someone could point me in the Right Direction.

Thank you

**Comment by ericblade** [comment]
I unfortunately don't know anything about Vue, so I'm not much help there, but there is a fairly popular and reasonably active vue-quaggajs module (though i believe it uses the original quagga, not this fork) https://github.com/sin-tanaka/vue-quagga

**Comment by GaryJurgens** [comment]
Hi

Thanks for the reply. I had a look at the one you mentioned, but it still seems to use Vue 2.

I am going to attempt a Vue 3 wrapper / interface for Quagga 2, I will upload it here if I am am able to come right.

Thanks Gary Jurgens

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by ericblade** [reply]
I think that's correct.  I would definitely welcome a Vue example, that we can put in /examples, or a separate repo we can link in README or whatever. :-)

**Reply by ericblade** [reply]
@adolfie450 did you get anywhere with that? :-)

**Reply by Joso997** [reply]
Any luck?

**Comment by ericblade** [comment]
https://github.com/DevinNorgarb/vue-quagga-2

updated fork that uses quagga2 with vue2, only reference i see to vue 3 with quagga is this post.
